test/src/comps/test_precond.cc

```diff
@@ -72,9 +72,8 @@ class Test_rpc_svd_sjlt : public ::testing::Test
         );
         
         // check the result
-        double reldtol = RELDTOL;
         EXPECT_EQ(rank, n);
-        double *ignore;
+        double *ignore = nullptr;
```

Comment:
Are you generally in favor of ``nullptr`` over ``NULL``, or do you determine which to use on a case-by-case basis?

Reply:
`NULL` is a macro defined to be the integer 0. `nullptr`, is a key word added to the language in C++11, and is preferred because it has a pointer type, so that it doesn't automatically convert to an integer. This resolves ambiguity in template deduction and overload resolution between int and pointer types.

Reply:
That said I will not complain as long as your pointers are initialized to zero, any method will do the trick.